### ISSUE TITLE
Backslash at end of comment incorrectly detected

### ISSUE THREAD

**Issue by labrys** [sobolevn/flake8-broken-line#36]
A backslash at the end of a comment is detected as a line-breaking slash.

```python
# A remote path can be entered.  e.g.:  \\server\
```
N400: Found backslash that is used for line braking
